rnndb/graph/g80_texture.xml

```diff
@@ -6,157 +6,312 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
+				<doc>the same as YUY2</doc>
+			</value>
+			<value value="0x24" name="DXT1"/>
+			<value value="0x25" name="DXT23"/>
+			<value value="0x26" name="DXT45"/>
+			<value value="0x27" name="DXN1"/>
+			<value value="0x28" name="DXN2"/>
+			<value value="0x10" name="BC6H_SF16" variants="GF100-">
+				<doc>similar to ZOH (FP16)</doc>
+			</value>
+			<value value="0x11" name="BC6H_UF16" variants="GF100-">
+				<doc>similar to ZOH (UFP16)</doc>
+			</value>
+			<value value="0x17" name="BC7U" variants="GF100-">
+				<doc>similar to ZOL (UNORM8)</doc>
+			</value>
+			<value value="0x06" name="ETC2_RGB"/>
+			<value value="0x0a" name="ETC2_RGB_PTA"/>
+			<value value="0x0b" name="ETC2_RGBA"/>
+			<value value="0x19" name="EAC"/>
+			<value value="0x1a" name="EACX2"/>
+
+			<value value="0x29" name="Z24S8">
+				<doc>stencil,    no VCAA,   24-bit fixed-point depth (legacy format)</doc>
+			</value>
+			<value value="0x2a" name="X8Z24">
+				<doc>no stencil, no VCAA,   24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2b" name="S8Z24">
+				<doc>stencil,    no VCAA,   24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2c" name="X4V4Z24__COV4R4V">
+				<doc>no stencil, 4+4 VCAA,  24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2d" name="X4V4Z24__COV8R8V">
+				<doc>no stencil, 8+8 VCAA,  24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2e" name="V8Z24__COV4R12V">
+				<doc>no stencil, 4+12 VCAA, 24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2f" name="ZF32">
+				<doc>no stencil, no VCAA,   32-bit floating-point depth</doc>
+			</value>
+			<value value="0x30" name="ZF32_X24S8">
+				<doc>stencil,    no VCAA,   32-bit floating-point depth</doc>
+			</value>
+			<value value="0x31" name="X8Z24_X20V4S8__COV4R4V">
+				<doc>stencil,    4+4 VCAA,  24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x32" name="X8Z24_X20V4S8__COV8R8V">
+				<doc>stencil,    8+8 VCAA,  24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x33" name="ZF32_X20V4X8__COV4R4V">
+				<doc>no stencil, 4+4 VCAA,  32-bit floating-point depth</doc>
+			</value>
+			<value value="0x34" name="ZF32_X20V4X8__COV8R8V">
+				<doc>no stencil, 8+8 VCAA,  32-bit floating-point depth</doc>
+			</value>
+			<value value="0x35" name="ZF32_X20V4S8__COV4R4V">
+				<doc>stencil,    4+4 VCAA,  32-bit floating-point depth</doc>
+			</value>
+			<value value="0x36" name="ZF32_X20V4S8__COV8R8V">
+				<doc>stencil,    8+8 VCAA,  32-bit floating-point depth</doc>
+			</value>
+			<value value="0x37" name="X8Z24_X16V8S8__COV4R12V">
+				<doc>stencil,    4+12 VCAA, 24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x38" name="ZF32_X16V8X8__COV4R12V">
+				<doc>no stencil, 4+12 VCAA, 32-bit floating-point depth</doc>
+			</value>
+			<value value="0x39" name="ZF32_X16V8S8__COV4R12V">
+				<doc>stencil,    4+12 VCAA, 32-bit floating-point depth</doc>
+			</value>
+			<value value="0x3a" name="Z16">
```

Comment:
Not clear - but I cannot find any trace of these before fermi, so let's go with GF100+